glide.yaml

```diff
@@ -1,7 +1,7 @@
 package: github.com/m3db/m3
 import:
 - package: github.com/m3db/m3x
-  version: 994ab3222cc45bdb1b439f96d37221651b24feca
+  version: 49a94422a02501a99c1ae6c1a632f83aea9efad6
```

Comment:
need to update glide.lock too

Reply:
Yeah was wondering why literally every single Semaphore test was failing.. Updated now though